<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGui::EditorDbQueryOptionsPage</class>
 <widget class="QWidget" name="QtGui::EditorDbQueryOptionsPage">
  <property name="windowTitle">
   <string>Metadata search</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox { font-weight: bold };</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="musicBrainzUrlLabel">
     <property name="text">
      <string>MusicBrainz URL</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QtUtilities::ClearLineEdit" name="musicBrainzUrlLineEdit">
     <property name="placeholderText">
      <string>https://musicbrainz.org/ws/2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="coverArtArchiveUrlLabel">
     <property name="text">
      <string>Cover Art Archive URL</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QtUtilities::ClearLineEdit" name="coverArtArchiveUrlLineEdit">
     <property name="placeholderText">
      <string>http://coverartarchive.org</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lyricWikiUrlLabel">
     <property name="text">
      <string>LyricWiki URL</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lyricWikiUrlLineEdit">
     <property name="placeholderText">
      <string>https://lyrics.fandom.com</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtUtilities::ClearLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">qtutilities/widgets/clearlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>musicBrainzUrlLineEdit</tabstop>
  <tabstop>lyricWikiUrlLineEdit</tabstop>
  <tabstop>coverArtArchiveUrlLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
